libs/payments/ui/src/lib/client/components/Header/en.tfl

```diff
@@ -0,0 +1,33 @@
+# Component - Header
+
+header-help
+  .title = Help
+  .aria-label = Help
+  .alt = Help
+
+bento-menu =
+  .title = Mozilla products # hover text for bento menu
+  .aria-label = Mozilla products # hover text for bento menu
+  .alt = Mozilla Logo # bento menu logo alt text
+
+
+bento-menu-close
+  .alt = Close  # Icon shown in close-menu button
+bento-menu-tagline = More products from Mozilla that protect your privacy
+
+bento-menu-firefox-desktop = { -brand-firefox } Browser for Desktop
```

Comment:
@bcolsson , I can't remember do localization ids need to be unique for the entire monorepo or is it just per service `ftl`, i.e. `payments-next.ftl` in this case?

Reply:
It's best practice to keep string IDs name spaced and globally unique for all l10n projects. This is especially true for FXA since `en.ftl` files ultimately get combined with other files into a single file during the string extraction process.

Reply:
Great, prefixed them with `payments-header-` and made sure all are unique. Thanks!